Dockerfile.ubuntu

```diff
@@ -1,4 +1,4 @@
-ARG OS_TAG=18.04
+ARG OS_TAG=20.04
```

Comment:
I wonder if that change could go upstream too.

Reply:
It's not needed anymore.